File: CPP/High_Voltage/cInterceptor.cpp
#include"GlobalVariables.h"

bool cInterceptor::Alive() {
    return cInterceptor::m_bAlive;
}

void cInterceptor::Alive(bool bNowAlive) {
    cInterceptor::m_bAlive = bNowAlive;
}

int cInterceptor::Row() {    
    return cInterceptor::m_iRow;
}

void cInterceptor::Row(int iNowRow) {
    cInterceptor::m_iRow = iNowRow;    
}

int cInterceptor::Column() {
    return cInterceptor::m_iColumn;
}

void cInterceptor::Column(int iNowCol) {
    cInterceptor::m_iColumn = iNowCol;    
}

File: HTML/High_Voltage_JS/cInterceptor.js

class cInterceptor {
   //Private
   #m_bAlive = true;
   #m_iRow;
   #m_iColumn;

   //Public
   Alive() {
      return cInterceptor.#m_bAlive;
   }

   Alive(bNowAlive) {
      cInterceptor.#m_bAlive = bNowAlive;
   }

   Row() {
      return cInterceptor.#m_iRow;
   }

   Row(iNowRow) {
      cInterceptor.#m_iRow = iNowRow;
   }

   Column() {
      return cInterceptor.#m_iColumn;
   }

   Column(iNowCol) {
      cInterceptor.#m_iColumn = iNowCol;
   }

}